pkg/controller/operators/catalog/operator_test.go

```diff
@@ -54,25 +54,35 @@ func TestTransitionInstallPlan(t *testing.T) {
 			Reason:  v1alpha1.InstallPlanReasonComponentFailed,
 			Message: errMsg,
 		}
+		// TODO(alecmerdler): Test `approval: Manual`
 	)
 	var table = []struct {
 		initial    v1alpha1.InstallPlanPhase
 		transError error
+		approval   v1alpha1.Approval
+		approved   bool
 		expected   v1alpha1.InstallPlanPhase
 		condition  *v1alpha1.InstallPlanCondition
 	}{
-		{v1alpha1.InstallPlanPhaseNone, nil, v1alpha1.InstallPlanPhasePlanning, nil},
-		{v1alpha1.InstallPlanPhaseNone, err, v1alpha1.InstallPlanPhasePlanning, nil},
+		{v1alpha1.InstallPlanPhaseNone, nil, v1alpha1.ApprovalAutomatic, false, v1alpha1.InstallPlanPhasePlanning, nil},
+		{v1alpha1.InstallPlanPhaseNone, err, v1alpha1.ApprovalAutomatic, false, v1alpha1.InstallPlanPhasePlanning, nil},
```

Comment:
just for completeness: `ApprovalAutomatic` + `approved = true` should go to planning as well